diff -ura package.orig/scripts/dkms.mkconf package.new/scripts/dkms.mkconf
--- package.orig/scripts/dkms.mkconf	2025-03-02 22:39:31.809717913 +0100
+++ package.new/scripts/dkms.mkconf	2025-03-02 22:41:43.524234679 +0100
@@ -26,6 +26,7 @@
 PACKAGE_CONFIG="${pkgcfg}"
 NO_WEAK_MODULES="yes"
 PRE_BUILD="configure
+  --enable-linux-experimental
   --disable-dependency-tracking
   --prefix=/usr
   --with-config=kernel
